group-coordinator/src/main/java/org/apache/kafka/coordinator/group/Group.java

```diff
@@ -57,10 +57,15 @@ public String toString() {
      */
     String stateAsString(long committedOffset);
 
+    /**
+     * @return The {{@link GroupType}}'s LowerCase String representation based on the committed offset.
+     */
+    String stateAsLowerCaseString(long committedOffset);
```

Comment:
It seems that we only access it from within the class so we could remove it from the interface.

Reply:
Done.